src/plugin/minimap.js

```diff
@@ -124,7 +124,7 @@ export default class MinimapPlugin {
             if (!document.body.contains(this.params.container)) {
                 ws.container.insertBefore(this.params.container, null);
             }
-
+            // how solve this dependency??
             if (this.wavesurfer.regions && this.params.showRegions) {
```

Comment:
Here, checking if there is regions plugin with this.wavesurfer.regions, forces to call the plugin "regions". In this way, extending regions plugin, it cannot be called with a custom name.
Any idea of how to solve this "name" dependency ?

Reply:
hi maria, maybe it's better to move the separate regions file to a new pull request and restrict this one to "Removed private methods of plugins and generalized plugins' access".

Reply:
> Here, checking if there is regions plugin with `this.wavesurfer.regions`, forces to call the plugin "regions". In this way, extending `regions` plugin, it cannot be called with a custom name.
> Any idea of how to solve this "name" dependency ?

Hi, we don't decide how to fix this